src/index.js

```diff
@@ -29,16 +28,10 @@ class FeathersError extends Error {
       message = msg;
     }
 
-    if (data) {
-      // NOTE(EK): To make sure that we are not messing
-      // with immutable data, just make a copy.
-      // https://github.com/feathersjs/feathers-errors/issues/19
-      newData = Object.assign({}, data);
-
-      if (newData.errors) {
-        errors = newData.errors;
-        delete newData.errors;
-      }
+    if (data && data.errors) {
```

Comment:
Can't we just clone the object using:

```
let clone = JSON.parse(JSON.stringify(data));
```
In contract to `Object.assign()` this works well with arrays.